proto/sf/tron/type/v1/block.proto

```diff
@@ -4,103 +4,82 @@ package sf.tron.type.v1;
 option go_package = "github.com/streamingfast/firehose-tron/pb/sf/tron/type/v1;pbtron";
 
 import "google/protobuf/any.proto";
-import "api/api.proto";
 import "core/Tron.proto";
 
 // Block represents a Tron block from the RPC response
 message Block {
+  // Block hash in bytes
   bytes id = 1;
   BlockHeader header = 2;
   repeated Transaction transactions = 3;
 }
 
 // BlockHeader represents the block header from the RPC response
 message BlockHeader {
+  // Block number (height)
   uint64 number = 1;
+  // Merkle root of all transactions in the block
   bytes tx_trie_root = 2;
+  // Address of the block producer (witness)
   bytes witness_address = 3;
   uint64 parent_number = 4;
   bytes parent_hash = 5;
   uint32 version = 6;
+  // Block timestamp in milliseconds since Unix epoch
   int64 timestamp = 7;
+  // Witness signature of the block
   bytes witness_signature = 8;
 }
 
 // Transaction represents a flattened Tron transaction
 message Transaction {
-  // Basic transaction info
-  // TODO validate what field we want here
+  // Transaction hash in bytes
   bytes txid = 1;
   repeated bytes signature = 2;
   bytes ref_block_bytes = 3;
   bytes ref_block_hash = 4;
+  // Transaction expiration timestamp in milliseconds
   int64 expiration = 5;
+  // Transaction creation timestamp in milliseconds
   int64 timestamp = 6;
 
-  // TODO Do we want result and result message here?
+  // Contract execution result in bytes
+  repeated bytes constant_result = 7;
 
-  // Raw TransactionExtention
-  protocol.TransactionExtention transaction_extention = 7;
+  bool result = 8;
+  // Transaction execution status code (see response_code enum)
+  response_code code = 9;
+  // Error message if any
+  bytes message = 10;
+  // Energy consumed by the transaction
+  int64 energy_used = 11;
+  // Energy penalty for the transaction
+  int64 energy_penalty = 12;
 
-  // Raw TransactionInfo
-  protocol.TransactionInfo transaction_info = 8;
+  // Raw TransactionInfo from the Tron protocol containing execution details
+  // source: https://buf.build/streamingfast/tron-protocol/file/main:core/Tron.proto#L453
+  protocol.TransactionInfo info = 13;
 
-  // TODO Add proper type for this
-  repeated protocol.Transaction.Contract contracts = 9;
+  // List of contracts executed in this transaction
+  // source: https://buf.build/streamingfast/tron-protocol/file/main:core/Tron.proto#L337
+  repeated protocol.Transaction.Contract contracts = 14;
```

Comment:
The original proto mention that this actually always contains a single contract and that the type is repeated for potential future extension, we should bring that comment in.

Reply:
also added that comment on signature cause it's the same thing.